src/python/create.py

```diff
@@ -578,6 +567,37 @@ def _get_relation(related_objs, cell_value, parsed_cell):
         return None
 
 
-def cellToBool(val):
+def is_cell_true(val):
     """Converts a cell value to a useable boolean"""
-    return True if str(val).lower() != "false" else False
+    return str(val).lower() != "false"
+
+
+def copyMaterial(material, new_project, project):
```

Comment:
beautiful function!
✅ great docstrings!
✅ great comments!
✅ clear self documenting code!

great job all around!

The only thing that I could think to make it even better would be if we had python typing and docstrings in either [Google](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) or [Pep ](https://peps.python.org/pep-0257/) style